packages/checkout/sdk/src/widgets/definitions/configurations/widget.ts

```diff
@@ -8,6 +8,12 @@ export type WidgetConfiguration = {
   /** The theme of the Checkout Widget (default: "DARK") */
   theme?: WidgetTheme;
   language?: WidgetLanguage;
+  /** Enable/disable Wallet Connect */
+  walletConnect?: {
+    enable: boolean;
+    includeWalletIds: string[];
+    excludeWalletIds: string[];
```

Comment:
Hey @mimi-imtbl , sorry can we remove these two arrays for now until we enable wallet connect whitelisting for partners